cis/system/hsp-cis-1-1-20-pki-key-restrict.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: hsp-cis-1-1-20-pki-key-restrict
+spec:
+  tags: ["CIS", "1.1.21", "K8s", "compliance"]
+  message: "This policy will restrict Kubernetes PKI keys"
+  nodeSelector:
+    matchLabels:
```

Comment:
It's not enforceable @AbhinavCSY

Reply:
Permission 600 can't set by KubeArmor Policy @raviknox